client/src/app/index.js

```diff
@@ -83,109 +84,107 @@ class App extends Component {
-                  path={ROUTES.COHORT_PROJECTS_PAGE}
-                  exact
-                  render={(props) => (
-                    <AdminProject {...props} logout={this.logout} />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_COHORT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="cohort"
-                      addLink="/api/v1/cohorts"
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_COHORT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="cohort"
-                      editLink={`/api/v1/cohorts/${props.match.params.cohortId}`}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_STUDENT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="student"
-                      addLink="/api/v1/alumni"
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_STUDENT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="student"
-                      editLink={`/api/v1/alumni/${props.match.params.studentId}`}
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.ADD_PROJECT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="project"
-                      addLink="/api/v1/projects"
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
-
-                <Route
-                  path={ROUTES.EDIT_PROJECT}
-                  exact
-                  render={(props) => (
-                    <AddEditForm
-                      {...props}
-                      formType="project"
-                      editLink={`/api/v1/projects/${props.match.params.projectId}`}
-                      cohortId={props.match.params.cohortId}
-                    />
-                  )}
-                />
+                <LogoutContext.Provider value={{ logout: this.logout }}>
+                  <Route
+                    exact
+                    path={ROUTES.HOME_PAGE}
+                    render={(props) => <Statistics {...props} />}
```

Comment:
Here there are no additional props you need to pass to `Statistics component` so it's better to use `component` instead of `render` and apply that for other components
```suggestion
                  component={Statistics} 
```